pydy/tests/test_system.py

```diff
@@ -395,3 +397,67 @@ def test_integrate(self):
         sys = System(self.kane, times=times)
         with testing.assert_raises(NotImplementedError):
             sys.generate_ode_function(generator='made-up')
+
+    def test_joint_system_init(self):
+        sys = System(method='joints')
+
+        # test that none of attributes set when system is called with kane
+        # are set.
+        assert hasattr(sys, 'method')
+        assert sys.method == 'joints'
+        assert sys.constants == dict()
+        assert sys.specifieds == dict()
+        assert sys.initial_conditions == dict()
+
+        # test all the attributes used to build system using joints are set.
+        assert isinstance(sys.ground, Ground)
+        assert sys.origin == sys.ground.origin
+        assert sys.gravity is None
+        assert isinstance(sys.reference_frame, ReferenceFrame)
+        assert sys._body_list == []
+
+    def test_joint_system_gravity(self):
+        sys = System(method='joints')
+        sys.add_gravity()
```

Comment:
Yup! I am removing all the redundant classed from utils and user can pass the symbol the gravity.